### PR TITLE
Adding FIPS job to MTC test run

### PR DESCRIPTION
**Pull request by aviavraham**
Implement rollout of FIPS-enabled OCP 4.17 testing for MTC test run

### CODE REVIEW

File: ci-operator/config/mtc-qe/mtc-e2e-qev2/mtc-qe-mtc-e2e-qev2-master__mtc1.8-ocp4.17-lp-interop.yaml [hunk 1]
```diff
@@ -75,6 +75,32 @@ tests:
     - ref: mtc-prepare-clusters
     - ref: mtc-execute-tests
     workflow: firewatch-cluster
+- as: mtc-interop-aws-fips
+  cron: 0 6 11 2 *
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      CLUSTER1_CONFIG: name-prefix=mtc-source;base-domain=cspilp.interop.ccitredhat.com;platform=aws;region=us-east-2;version=4.17;stream=stable;worker-flavor=m5.xlarge;timeout=2h;fips_enabled=true
```
Comment (calebevans) [inline ci-operator/config/mtc-qe/mtc-e2e-qev2/mtc-qe-mtc-e2e-qev2-master__mtc1.8-ocp4.17-lp-interop.yaml:83]:
```suggestion
      CLUSTER1_CONFIG: name-prefix=mtc-source;base-domain=cspilp.interop.ccitredhat.com;platform=aws;region=us-east-2;version=4.17;stream=stable;worker-flavor=m5.xlarge;timeout=2h;fips=true
```

File: ci-operator/config/mtc-qe/mtc-e2e-qev2/mtc-qe-mtc-e2e-qev2-master__mtc1.8-ocp4.17-lp-interop.yaml [hunk 2]
```diff
@@ -75,6 +75,32 @@ tests:
...
+      CLUSTER2_CONFIG: name-prefix=mtc-target;base-domain=cspilp.interop.ccitredhat.com;platform=aws;region=us-east-2;version=4.17;stream=nightly;worker-flavor=m5.xlarge;timeout=2h;fips_enabled=true
```
Comment (calebevans) [inline ci-operator/config/mtc-qe/mtc-e2e-qev2/mtc-qe-mtc-e2e-qev2-master__mtc1.8-ocp4.17-lp-interop.yaml:84]:
```suggestion
      CLUSTER2_CONFIG: name-prefix=mtc-target;base-domain=cspilp.interop.ccitredhat.com;platform=aws;region=us-east-2;version=4.17;stream=nightly;worker-flavor=m5.xlarge;timeout=2h;fips=true
```

**Review by calebevans — CHANGES_REQUESTED** [review]
I believe the variables for this workflow are just `fips` rather than `fips_enabled`. Also, please run a rehearsal on this one to verify it works.

**Comment by aviavraham** [comment]
Hi @calebevans 
As you suggested, I made the change to the YAML file. Could you please review the code again?

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.17-lp-interop-mtc-interop-aws-fips

**Comment by calebevans** [comment]
@aviavraham if this rehearsal works, we can merge. I just want to make sure that works.

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.17-lp-interop-mtc-interop-aws-fips

**Comment by calebevans** [comment]
Waiting on https://github.com/RedHatQE/openshift-cli-installer/pull/677 to be merged to fix IPI FIPS installs

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.17-lp-interop-mtc-interop-aws-fips

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.17-lp-interop-mtc-interop-aws-fips

**Comment by dfrazzette** [comment]
Rehearsal run is successfully creating/installing the FIPS cluster; now failing with the same blocking errors seen in [weekly periodic jobs](https://prow.ci.openshift.org/view/gs/test-platform-results/logs/periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.17-lp-interop-mtc-interop-aws/1815269464950706176). Ready to merge.

**Pull request merged**